src/tracking.F90

```diff
@@ -59,6 +59,9 @@ subroutine transport(p)
       micro_xs % last_E = ZERO
     end if
 
+    ! Set xs_seed to be current tracking prn seed
+    xs_seed = prn_seed(STREAM_TRACKING)
```

Comment:
With the `STREAM_URR_PTABLE` we can remove this line and the one at line 238.  It will be handled by `set_particle_seed`